test/test-main.cc

```diff
@@ -27,12 +24,12 @@ int main(int argc, char** argv) {
   // doesn't help.
   SetErrorMode(SEM_FAILCRITICALERRORS | SEM_NOGPFAULTERRORBOX |
                SEM_NOOPENFILEERRORBOX);
-#endif
   // Disable message boxes on assertion failures.
   _CrtSetReportMode(_CRT_ERROR, _CRTDBG_MODE_FILE | _CRTDBG_MODE_DEBUG);
   _CrtSetReportFile(_CRT_ERROR, _CRTDBG_FILE_STDERR);
   _CrtSetReportMode(_CRT_ASSERT, _CRTDBG_MODE_FILE | _CRTDBG_MODE_DEBUG);
   _CrtSetReportFile(_CRT_ASSERT, _CRTDBG_FILE_STDERR);
+#endif
```

Comment:
Different conditions in ``#if``.

Line 16:
```
#ifdef _MSC_VER
```

Line 24:
```
#ifdef _WIN32
```

Reply:
OK, changed.